backend/go.mod

```diff
@@ -5,30 +5,31 @@ go 1.12
 require (
 	github.com/coreos/go-oidc v2.2.1+incompatible
 	github.com/emicklei/go-restful/v3 v3.10.1 // indirect
-	github.com/felixge/httpsnoop v1.0.3 // indirect
 	github.com/fsnotify/fsnotify v1.6.0
 	github.com/go-openapi/swag v0.22.3 // indirect
+	github.com/go-playground/validator/v10 v10.12.0
 	github.com/gobwas/glob v0.2.3
 	github.com/google/gnostic v0.6.9 // indirect
-	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/handlers v1.5.1
 	github.com/gorilla/mux v1.8.0
-	github.com/imdario/mergo v0.3.13 // indirect
+	github.com/gorilla/schema v1.2.0
```

Comment:
We're adding a dependency which is unmaintained here. I haven't looked into why this has been added or what alternatives there may or may not be.

Reply:
Let's remove this when we get rid of gorilla.